Mathlib/Algebra/Algebra/Basic.lean

```diff
@@ -243,6 +244,11 @@ instance (priority := 99) Semiring.toNatAlgebra : Algebra ℕ R where
 instance nat_algebra_subsingleton : Subsingleton (Algebra ℕ R) :=
   ⟨fun P Q => by ext; simp⟩
 
+@[simp]
+lemma algebraMap_comp_nat_cast_eq_cast (R A : Type*) [CommSemiring R] [Semiring A] [Algebra R A] :
```

Comment:
I think we're converging to this convention, see e.g. many lemmas with `cardinalMk` in the names:
```suggestion
lemma algebraMap_comp_natCast_eq_cast (R A : Type*) [CommSemiring R] [Semiring A] [Algebra R A] :
```